activities/codebase-stewardship/for-existing-projects.md

```diff
@@ -30,8 +30,8 @@ We also need the developing organization to give us:
 We will:
 
 * conduct [audits](../codebase-auditing/index.md) to help comply with the standard on every pull request
-* provide training and workshops to develop better reusable codebases as well as working with the community
-* run community events and calls to integrate a community
+* provide [training and workshops](../workshops/index.md) to develop better reusable codebases as well as working with the community
```

Comment:
This could link could be split onto two to link to ..trainings/index.md as well.